src/resmom_win/start_exec.c

```diff
@@ -1997,8 +1998,11 @@ finish_exec(job *pjob)
 			/* Need to launch a shell*/
 			launch_shell = 1;
 		}
+		/* If we fail to get cmd shell(unlikely), use "cmd.exe" as shell */
+		if (0 != get_cmd_shell(cmd_shell, sizeof(cmd_shell)))
```

Comment:
It's not illegal to use (0 != foo()), but we use (foo() != 0) virtually everywhere in the code.